package-lock.json

```diff
@@ -1,9797 +1,8 @@
 {
```

Comment:
This file changed a lot, and the version went backwards.  Is that a problem?

Reply:
That's kind of a ton of deletions. That seems strange, but likely ok. It doesn't seem relevant to this PR, though, as you're not updating `package.json` dependencies, so I think removing it would be best.

Reply:
There's still a lot of diffs, but maybe those came from Bill's feature branch.

Reply:
Fixed. See below.